db/migrate/20201028103608_create_project_folders.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class CreateProjectFolders < ActiveRecord::Migration[6.0]
+  include DatabaseHelper
+
+  def up
+    create_table :project_folders do |t|
```

Comment:
I would call the table and model just `folder(s)`. 
If we introduce folders on more levels, probably it will become STI, and a base class can be `folder`, ProjectFolder, TaskFolder, etc could be subclasses.

Reply:
It will be only for projects